driver-core/src/main/java/com/datastax/driver/core/AggregateMetadata.java

```diff
@@ -225,15 +227,17 @@ public KeyspaceMetadata getKeyspace() {
     }
 
     /**
-     * Returns the full name of this aggregate.
+     * Returns the CQL signature of this aggregate.
      * <p>
      * This is the name of the aggregate, followed by the names of the argument types between parentheses,
      * like it was specified in the {@code CREATE AGGREGATE...} statement, for example {@code sum(int)}.
      *
      * @return the full name.
      */
-    public String getFullName() {
-        return fullName;
+    public String getSignature() {
```

Comment:
I'm removing the old "getFullName()" method as discussed previously, but providing an alternate "getSignature()" method. The goal is to provide here a valid CQL string representing the signature of the function/aggregate. This can be used by client code as an unique identifier for the function (although we do not use it for this purpose, see later). Note that for it to be valid, function names and parameters names/types are all quoted if necessary. And lastly, note that signatures must not contain the "frozen" keyword as the outermost element (i.e. `frozen<foo>` is invalid, but `list<frozen<set<text>>>` is valid).